javascript/javascript2/week1/danish_letters.js

```diff
@@ -0,0 +1,35 @@
+const danishLetters = ["å", "æ", "ø"];
+
+function getDanishLetters(danishString) {
+  let letters = danishString.split("");
+
+  let resultArray = [];
+  for (let j = 0; j < 3; j++) {
+    for (let i = 0; i < letters.length; i++) {
+      if (letters[i] === danishLetters[j]) {
+        resultArray.push(letters[i]);
+      }
```

Comment:
This block has time complexity of O(nˆ2), which is not good. if the block use `Set` as the lookup table of danish letter, it can achieve O(n).

Reply:
So thanks for comments. specially use set 👍